bundles/org.openhab.io.mqttembeddedbroker/pom.xml

```diff
@@ -41,5 +41,48 @@
       <version>1.4.199</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-common</artifactId>
+      <version>4.1.34.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-buffer</artifactId>
+      <version>4.1.34.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport</artifactId>
+      <version>4.1.34.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec</artifactId>
+      <version>4.1.34.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-mqtt</artifactId>
+      <version>4.1.34.Final</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-resolver</artifactId>
+      <version>4.1.34.Final</version>
```

Comment:
Would it make sense to extract the repeated version number? Is it expected that we need to add all of them here?

Reply:
Done. As I said: I don't fully understand why it is necessary to add them here as the build didn't require it before splitting the features into seperate files. Maybe it depends on the build order which version is downloaded.

All of these dependencies were declared in the feature before, I only added them to the POM to pin the downloaded version.